content/en-us/tutorials/core/building/create-basic-visual-effects.md

```diff
@@ -42,7 +42,7 @@ To create a flare:
       </thead>
       <tbody>
       <tr>
-      <td>VFX_Flare</td>
+      <td>Emitter_Flare</td>
```

Comment:
Hey @dannywalter! You can name the part that represent the area of where you want your particles to emit from to any name that will help you remember its functionality. The tutorial chooses to call this part VFX_Flare so that the actual particle emitter can be called Emitter_Flare. I'll add some updates to the tutorial that will make this more clear, thank you for the feedback!

Reply:
You make an excellent point that we incorrectly instruct readers to rename this part to Emitter_Flare. My apologies, I will fix this right away.

Reply:
Ahhh, I see what's going on now.

Looking at the actual place file, I see it wasn't only to _not instruct_ readers to name the parent part VFX_Flare to Emitter_Flare. We also need _to instruct_ readers to name the Particle Emitter itself Emitter_Flare.

`59  - 1. Add a particle emitter to this attachment. Name it **Emitter_Flare**.`

Reply:
That's right! You can definitely rename the parent part or the particle emitter itself anything that you'd like, but the naming recommendation in this tutorial aids in scanning the Workspace. Some games will have TONS parent part "systems", so knowing their function can be incredibly helpful in finding what you're looking for.

Reply:
Agreed! Now that I understand the _actual_ naming convention after seeing the place file itself, I like it much better than what I was confusingly following in the tutorial!

So lines 45 and 159 do not need to be changed from "VFX_" to "Emitter". Yay.

All that's left is:
- Lines 53 and 169 - Remove instruction incorrectly naming the block parts to Emitter_Flare and Emitter_Dustmotes
- Lines 59 and 174 - Add instruction to name the particle emitters themselves Emitter_Flare and Emitter_Dustmotes, which isn't present and what causes the most confusion.

59  - 1. Add a particle emitter to this attachment. Name it **Emitter_Flare**.
174 - 1. Add a particle emitter to this part. Name it **Emitter_Dustmotes**.

Reply:
That's right, and those changes are currently live :)